### ISSUE TITLE
md-tab inner md-dialog: md-tab-item always set max-width=0px

### ISSUE THREAD

**Issue by tsdragon0404** [angular/material#4896]
https://codepen.io/anon/pen/pjNVjv
![untitled](https://cloud.githubusercontent.com/assets/11863784/10157985/0afc1474-66b9-11e5-8b8c-72d91d596b7d.png)

**Comment by matitalatina** [comment]
+1

**Comment by filipSu** [comment]
+1

**Comment by GrafRaf** [comment]
+1 and not only in inner md-dialog,  ( v. 0.11.1 )
with version 0.11.0 -  all work fine

**Comment by Lorthirk** [comment]
+1

**Comment by tommy8694** [comment]
+1
same issue in sidenav

**Comment by relevantmedia** [comment]
I tried to add md-tabs inside a sidenav. They are broken here as well. +1

**Comment by Mathem8tic** [comment]
http://codepen.io/Mathem8tic/pen/JYNQBV

I am also having issues using v 0.11.2, it seems to happen when `<md-tab ... md-no-pagination>` is used.  

Are tabs not intended for use within the `<md-sidenav>`

**Comment by matitalatina** [comment]
Maybe you already know it, but this is my ugly-as-hell workaround to fix tabs inside sidenav:
```
md-sidenav md-tabs md-tabs-wrapper md-tabs-canvas md-tab-item.md-tab {
    max-width: inherit !important;
  }
```

**Comment by ddennis** [comment]
+1 - thanks for the "ugly-as-hell workaround" , it works

**Comment by datnordstrom** [comment]
1+

**Comment by Mathem8tic** [comment]
This issue seems to be fixed in v 11.4.

[CHANGELOG](https://github.com/angular/material/blob/master/CHANGELOG.md)

**Comment by pasqLisena** [comment]
Is this a duplicate of #4873?

**Comment by lmcorbo** [comment]
I still having this issue with v1.0.6, after closing and reopen the dialog that contains the tabs

**Comment by DevVersion** [comment]
@lmcorbo Can you confirm the issue is still present, when using `master` / HEAD?

**Comment by lmcorbo** [comment]
@DevVersion I test with master/head and can't reproduce the issue, so i think that is resolved

**Comment by DevVersion** [comment]
Perfect, thanks for the confirm. That means the the PR which fixes that, is assigned to be a major change, and it will be released within `1.1.0`
